Makefile

```diff
@@ -55,7 +55,7 @@ PKG_MAN_OPTS ?= $(PKG_FROM_VERSION) $(PKG_CHANNELS) $(PKG_IS_DEFAULT_CHANNEL)
 # Image URL to use all building/pushing image targets
 IMG ?= quay.io/submariner/submariner-operator:$(VERSION)
 # Produce CRDs that work back to Kubernetes 1.11 (no version conversion)
```

Comment:
This needs to be changed too ;-).

Reply:
the IMG ? I use it to replace the placeholder in the CSV file

Reply:
I’d picked line 57, not 56 — the “Produce CRDs that work back to Kubernetes 1.11 (no version conversion)” is wrong now.

Reply:
wow good catch! +1